docs/guides/managed-execution.md

```diff
@@ -0,0 +1,133 @@
+---
+description: Prefect will run your deployment on our infrastructure.
+tags:
+    - managed infrastructure
+search:
+  exclude: true
+---
+
+# Managed Execution
+
+Prefect Cloud provides a **prefect-managed** work pool that you can use for your workflow execution environment.
+Deployments run with this work pool do not require a worker and do not require a cloud provider account.
+Prefect handles the infrastructure for you.
+
+Managed execution is a great option for users who want to get started quickly, with no infrastructure setup.
+
+!!! warning "Managed Execution is in alpha"
+    Managed Execution is currently in alpha.
+    Features are likely to change without warning.
+
+## Usage guide
+
+Run a flow with managed infrastructure in three steps.
+
+### Step 1
+
+Create a new work pool of type **prefect-managed**.
+
+!!! note
+    In the alpha period, you must have the managed execution feature enabled for your account to create a work pool of this type.
+
+### Step 2
+
+Create a deployment using the flow `deploy` method or `prefect.yaml`.
+
+Specify the name of your managed work pool, as shown in this example that uses the `deploy` method:
+
+```python hl_lines="9" title="managed-execution.py"
+from prefect import flow
+
+if __name__ == "__main__":
+    flow.from_source(
+    source="https://github.com/desertaxle/demo.git",
+    entrypoint="flow.py:my_flow",
+    ).deploy(
+        name="test-managed-flow",
+        work_pool_name="prefect-managed",
+    )
+```
+
+With your CLI authenticated to your Prefect Cloud workspace, run the script to create your deployment:
+
+<div class="terminal">
+```bash
+python managed-execution.py
+```
+</div>
+
+Note that this deployment uses flow code stored in a GitHub repository.
+If you would like to use your own flow code, push your code to your repository and specify your own repository URL and entrypoint.
+
+### Step 3
+
+Schedule a deployment run from the UI or from the CLI.
+
+It may take a minute for the flow run to start and the logs to appear in the UI.
+
+That's it! You ran a flow on remote infrastructure without any infrastructure setup, worker, or cloud provider account.
+
+### Adding dependencies
+
+You can install Python package dependencies at runtime by passing `job_variables={"env": {"EXTRA_PIP_PACKAGES": ["pandas", "prefect-aws"] }}` like this:
```

Comment:
Is it worth mentioning the pull_step to do this or should we not even get into it? Theres no benefit that I know of from doing that vs this, except you already have your requirements file

Reply:
I'll add that it's an alternative but not show it.